paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlTypeUtils.java

```diff
@@ -152,6 +152,11 @@ public static DataType toDataType(
             Boolean tinyInt1ToBool) {
         switch (type.toUpperCase()) {
             case BIT:
+                if (length == null || length == 1) {
+                    return DataTypes.BOOLEAN();
```

Comment:
Why cast bit(1) to boolean type?

Reply:
we refer to flink cdc , be consistent with flink cdc

https://ververica.github.io/flink-cdc-connectors/master/content/connectors/mysql-cdc.html#data-type-mapping.